riva/clients/tts/riva_tts_client.cc

```diff
@@ -127,6 +142,30 @@ main(int argc, char** argv)
 
   request.set_sample_rate_hz(rate);
   request.set_voice_name(FLAGS_voice_name);
+  if (not FLAGS_zero_shot_audio_prompt.empty()) {
+    auto zero_shot_data = request.mutable_zero_shot_data();
+    std::vector<std::shared_ptr<WaveData>> audio_prompt;
+    LoadWavData(audio_prompt, FLAGS_zero_shot_audio_prompt);
+    if (audio_prompt.size() != 1) {
+      LOG(ERROR) << "Unsupported number of audio prompts. Need exactly 1 audio prompt."
+        << std::endl;
+      return -1;
+    }
+    zero_shot_data->set_audio_prompt(&audio_prompt[0]->data[0], audio_prompt[0]->data.size());
+    auto zero_shot_rate = FLAGS_zero_shot_sample_rate;
+    if (FLAGS_zero_shot_encoding.empty() || FLAGS_zero_shot_encoding == "pcm") {
```

Comment:
even encoding can be picked from audio_prompt[0]->encoding

Reply:
Done